### PR TITLE
oadp-1.1: Fix oadp operator image tag

### PR DESCRIPTION
**Pull request by dmrost**
This fixes "unknown flag: --fs-backup-timeout error"

### CODE REVIEW

**Review by kaovilai — DISMISSED** [review]
/lgtm

**Review by shubham-pampattiwar — DISMISSED** [review]

**Comment by mateusoliveira43** [comment]
/hold
Should we update the config files and then run `make bundle` to create the bundle files?

**Comment by kaovilai** [comment]
> Should we update the config files and then run make bundle to create the bundle files?

ya. will push here in a bit.

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml
Comment (mateusoliveira43) [inline bundle/manifests/oadp-operator.clusterserviceversion.yaml:1]: Thanks for the PR @dmrost 

The files under `bundle/` folder are generated running `make bundle`. Can you please edit the files that hold the change, run `make bundle` and commit it? (I think @kaovilai can do this for you)

I believe the files you want to modify are https://github.com/openshift/oadp-operator/blob/oadp-1.1/Makefile#L117 and https://github.com/openshift/oadp-operator/blob/oadp-1.1/config/manifests/bases/oadp-operator.clusterserviceversion.yaml#L8

**Review by mateusoliveira43 — APPROVED** [review]

**Pull request merged**